package.json

```diff
@@ -0,0 +1,86 @@
+{
+    "name": "chowly",
```

Comment:
Maybe we should pick another name and also use the namespace.
```suggestion
    "name": "@goparrot/chowly-api-client",
```

Reply:
was inspired by google, they have almost the same naming for this kind of features

Reply:
Then to follow the convention of @goparrot packages, we can use `@goparrot/chowly`.